_includes/footer.html

```diff
@@ -107,21 +107,102 @@ <h2><a {% if column.new_window %}target="_blank" {% endif %}
   <script src="https://cdn.jsdelivr.net/npm/clipboard@2/dist/clipboard.min.js"></script>
   <script src="{{ site.baseurl }}/js/vanilla-tilt.min.js"></script>
   <script>
-	function resetCopyText(element) { /* function logic */ }
+
+	function resetCopyText(element) {
+		let childElements = element.childNodes
+		if (childElements.length > 3) {
+			childElements[3].innerHTML = "Copy to clipboard";
+		} else {
+		childElements[1].innerHTML = "Copy URL";
+		childElements[1].style.color = 'white';
+		childElements[1].style.background = "#00b39fff";
+        }
+	}
```

Comment:
Please correct the formatting and confirm if the added logic is same as previous. Thanks!